community/logging/src/test/java/org/neo4j/logging/RotatingFileOutputStreamSupplierTest.java

```diff
@@ -81,6 +82,8 @@
 
     @Rule
     public final TestDirectory testDirectory = TestDirectory.testDirectory( getClass(), fileSystem );
+    @Rule
+    public final SuppressOutput suppressOutput = SuppressOutput.suppressAll();
```

Comment:
That is because this test produce console output that is not really useful to see that each time.